tests/src/test/java/alluxio/shell/command/CopyToLocalCommandTest.java

```diff
@@ -35,6 +46,18 @@ public void copyToLocalDir() throws IOException, AlluxioException {
     fileReadTest("/testDir/foo/foobar2", 20);
     fileReadTest("/testDir/bar/foobar3", 30);
     fileReadTest("/testDir/foobar4", 40);
+    mOutput.reset();
```

Comment:
Our unit testing philosophy is that each test should focus on testing just one element of functionality. The first half of this test is testing copying a directory to a directory, while the second half is testing copying a relative file path to a directory. Could we split into two tests?